src/components/SharedComponents/Tabs.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import { Button } from "react-native";
+import colors from "styles/tailwindColors";
+
+type Tab = {
+  id: string,
+  text: string,
+  onClick: ( ) => void
+}
+
+type Props = {
+  tabs: Tab[],
+  activeId: string,
+}
+
+const Tabs = ( { tabs = [], activeId }: Props ): Node => (
+  <View className="bg-white flex flex-row">
+    {
+        tabs.map( ( { id, text, onClick } ) => {
+          const active = activeId === id;
+          const borderClass = `${active ? "bg-primary" : "bg-white"} h-1 rounded-t-lg`;
+          return (
+            <View key={id} className="flex-1">
+              <Button
+                onPress={onClick}
+                title={text}
+                color={active ? colors.primary : colors.grayText}
+                accessibilityLabel={text}
+              />
```

Comment:
There is the possibility to give [accessibilityRole](https://reactnative.dev/docs/accessibility#accessibilityrole) of "tab". But I could not find out yet if that is also best practice to use for tabs that are inside the screen and not at the bottom navigator. Anyway, a role of "button" or "link" would not make accessible the behaviour of the app when tapping this, so I would like to see a role added here that is best practice.